Ampeg Manual Tests.txt

```diff
@@ -0,0 +1,83 @@
+Test Suite: Ampeg.com Manual Tests
+Pre-Conditions: Before each test, Open ampeg.com
+Environment: Using in Chrome (fullscreen 1920x1080) on a laptop running Windows 10 in the US 
+
+Test ID: AMT_001
+Title: IsMyInfoSold
+Summary: Find out if the site sells personal information (1 page)
```

Comment:
I wouldn't use 'Find Out'. Write the test from the perspective that you know what the site does. The test is proving that functionality.

Summary: Verify that the site is not selling personal information.